### PR TITLE
S ct2

### PR DESCRIPTION
**Pull request by s-luresz**
Collected examin regulation pdf files and creating meta data file to each one. 
Created a function, which converts pdf to txt.

### CODE REVIEW

File: readFromPdf.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    # closing the pdf file object
+    pdfFileObj.close()
+
+    #write the text from pdf to txt file
+    output = open(txtname, 'w')
+    output.write(fileText)
+
+#test variables
+pdfTestName = input('Enter the pdf file name: ') #'examination_regulations/GPO-BMS_Lesefassung_AendS-2021-2.pdf'
```
Comment (MarioStanke) [inline readFromPdf.py:32]: Bitte den Kommentar verkuerzen, z.B. # path/to/my.pdf

**Review by MarioStanke — CHANGES_REQUESTED** [review]
The converted text files are missing:
sCt2: convert pdf to text, e.g. ASCII (e.g. pdftotext) ( 3 steps)

Better make pull requests for each task separately. Your PDFs had not been merged yet.

Please make it so that no manual work is required for each PDF. Implement a loop over your function to process a list of PDFs.

**Comment by s-luresz** [comment]
- The spaces around the character '=' are added in the code.

- The comment about the example file path was with the input deleted, because the program checks itself if there are any pdf files so the manual input of the name of pdf files is not neccessary. 
It was assumed that the pdf files that should be converted are in the directory 'data/examination_regulations'.

- Converted files (the txt ones) are added
- the loop over all files has been added (see point 2)

**Comment by s-luresz** [comment]
I saw that only the first page of pdf was converted. The bug is fixed and now all pages of pdf are converted.

File: readFromPdf.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    #write the text from pdf to txt file
+    output = open(txtname, 'w')
+    output.write(fileText)
+
+#convert pdf files in a dictionary into txt files
+directory = 'data/examination_regulations'
+
+for filename in os.listdir(directory):
+    f = os.path.join(directory, filename)
+    # checking if it is a file and if it is '.pdf' at the end
```
Comment (MarioStanke) [inline readFromPdf.py:39]:
```suggestion
    # checking if it is a file and if it ends with '.pdf'
```

**Review by MarioStanke — APPROVED** [review]
Looks good.

**Pull request merged**